yt/frontends/cholla/io.py

```diff
@@ -0,0 +1,55 @@
+import h5py
+
+from yt.utilities.io_handler import BaseIOHandler
```

Comment:
This is breaking some tests
```suggestion
from yt.utilities.io_handler import BaseIOHandler
from yt.utilities.on_demand_imports import _h5py as h5py
```